### DISCUSSION TITLE
Welcome to Dark-by-DM Discussions!

### DISCUSSION THREAD

**Discussion by DM164** [DM164/Dark-by-DM#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DM164** [comment]
I also invite you to join our Discord!
https://discord.gg/dNEYsW5pFM

**Comment by CharCharli** [comment]
![Screenshot (287)](https://user-images.githubusercontent.com/83393350/116515228-30423a80-a8f6-11eb-8fa9-c45d6c4249c5.png)
If i turned the theme off, my gmail will be turned into this, i cant find reason
